src/AddressRegistry.Projections.Syndication/appsettings.json

```diff
@@ -44,7 +44,12 @@
     "Building": "https://legacy.gebouw.staging-basisregisters.vlaanderen/v1/gebouwen/sync?embed=true",
     "BuildingAuthUserName": "x",
     "BuildingAuthPassword": "x",
-    "BuildingPollingInMilliseconds": 5000
+    "BuildingPollingInMilliseconds": 5000,
+
+    "Address": "https://legacy.adres.staging-basisregisters.vlaanderen/v1/adressen/sync?embed=true",
```

Comment:
I think we still need to replace `embed=true` with event/object?

Reply:
already done see `Program.cs` line 183-184